resources/todomvc/architecture-examples/react/webpack.common.js

```diff
@@ -0,0 +1,36 @@
+const path = require("path");
+
+module.exports = {
+    entry: {
+        app: "./src/index.js",
+    },
+    output: {
+        filename: "[name].bundle.js",
+        path: path.resolve(__dirname, "dist"),
+        clean: true,
+    },
+    resolve: {
+        extensions: [".js", ".jsx"],
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(js|jsx)$/,
+                exclude: /nodeModules/,
+                use: {
+                    loader: "babel-loader",
+                    options: {
+                        presets: [
+                            ["@babel/preset-env", { targets: "defaults" }],
+                            ["@babel/preset-react", { runtime: "automatic" }],
+                        ],
```

Comment:
I see it's already in `babel.config.js`, is it necessary to have both?
Also it's not clear to me what environement babel targets here... It looks like it targets some recent version of JS but it would be good to explicit this, especially that [babel seems to say otherwise](https://babeljs.io/docs/options#no-targets)...

Reply:
thanks - I removed the babel.config file